src/chrome/content/rules/LMU-Muenchen-IFI.xml

```diff
@@ -0,0 +1,48 @@
+<!--
+	Ludwig-Maximilians-Universitaet Muenchen
+	department: Institute of Informatics / Institut fuer Informatik
+  	
+	Ruleset for main page, see LMU-Muenchen.xml
```

Comment:
Please also add a reference to this rule on `LMU-Muenchen.xml`.

Reply:
Ok. I added a reference in #13469 to avoid merge conflict.